problem_builder/templates/html/swipeblock.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n %}
+{% if not hide_header %}
+<h4 class="question-title" id="heading_{{ self.html_id }}">{{ self.display_name_with_default }}</h4>
+{% endif %}
+{% if self.img_url.strip %}
+<img src="{{ self.expand_img_url }}" alt="" />
```

Comment:
@kaizoku This still doesn't work correctly. I think it's because the property is named `expanded_img_url`, not `expand_img_url`.